### ISSUE TITLE
`aria-hidden="true"` prevents screen reader users from accessing modals

### ISSUE THREAD

**Issue by digilou** [archivesspace/archivesspace#1673]
## Expected Behavior

When a user wants to make a request by clicking the "Request" button, a modal appears with a Request form. The expected behavior is that all users, including screen reader users, will be able to access the form content and fill out the form.

![image](https://user-images.githubusercontent.com/8865558/64924477-65396200-d791-11e9-8413-c2f58ede54eb.png)

## Current Behavior

The problem is that a screen reader user cannot hear anything inside of the modal because `aria-hidden="true"` has been added to the wrapping `div`. They cannot access the content because they have no idea that content exists.

https://github.com/archivesspace/archivesspace/blob/2bce49707d58603b6f4890628290c5b301bb7e0d/public/app/views/shared/_modal.html.erb#L2

This same problem also exists on the div wrapper for the Citation modal.

## Possible Solution

Remove `aria-hidden="true"` from the parent `div` within the shared _modal.html.erb file. 

This directly affects div#request_modal and div#cite_modal.

## Steps to Reproduce (for bugs)

1. Visit any ArchivesSpace public site. e.g. [Alaska Historical Collections](https://alaska.libraryhost.com/)
2. Choose an item from digital materials.
![image](https://user-images.githubusercontent.com/8865558/64924346-0a533b00-d790-11e9-8a05-2a12012796b2.png)
3. Turn on a screen reader to use for testing e.g. NVDA or VoiceOver
3. Navigate to and click on the Request button.
4. Listen for the content of the open dialog. I heard nothing when using NVDA.

## Context

At the Alaska State Library, we value feedback from our patrons. If we provide them with a request form, we'd like them to be able to independently use that form rather than forcing them into a phone call. By blocking access to the Request form, it denies our screen reader user patrons equal access to this service.

## Your Environment

* Version used: unknown, since it's hosted for us
* Environment name and version (e.g. Chrome 39, node.js 5.4): Firefox 69.0, NVDA 2019.2
* Operating System and version (desktop or mobile): Windows 10 desktop
* Link to your project: https://alaska.libraryhost.com/

**Comment by lmcglohon** [comment]
@digilou I was unable to replicate this issue with the latest release.

**Comment by digilou** [comment]
@lmcglohon Ok. I’ll check our version tomorrow at work to see if anything has changed. It’s been awhile since I’ve looked!

**Comment by digilou** [comment]
@lmcglohon Thanks for your response. This issue can likely close. Looks like my next step is talking to our vendor to update our version to resolve this issue.
